song_match/song/note.py

```diff
@@ -12,10 +12,11 @@
 class Note:
     __is_mixer_initialized = False
 
-    def __init__(self, note: str):
+    def __init__(self, note: str, duration: int):
```

Comment:
You could default the duration to a quarter note here, that way you don't always pass in a duration when constructing `Note` instances.

Reply:
Ok cook, went ahead and did that. Thanks